src/styles/blocks/_main-block.scss

```
@@ -0,0 +1,107 @@
+.main {
+
+  &__benefits {
+    --container-padding: 0 30px;
+    margin: 0 auto;
+    max-width: 1060px;
+    padding: var(--container-padding);
+
+    @include onTablet {
+      --container-padding: 0 34px;
+      margin-bottom: 140px;
+      gap: 65px 20px;
+    }
+
+    @include onDesktop {
+      --container-padding: 0;
+      max-width: 100%;
+
+      margin-bottom: 170px;
+      gap: 0 30px;
+    }
+  }
+
+  &__presentation {
+    padding: 0 12px;
+    margin-bottom: 110px;
+
+    @include onDesktop {
+      margin: 0 auto;
+      padding-bottom: 170px;
+    }
+  }
+
+  &__content-part {
+    --contentMrgnBot: 116px;
+
+    padding: 0 30px;
+    margin-bottom: var(--contentMrgnBot);
+
+    @include onTablet {
+      --contentMrgnBot: 140px;
+    }
+
+    @include onDesktop {
+      --contentMrgnBot: 170px;
+      padding: 0;
+    }
+  }
```

I'm not sure. Why do you use variable here?

```suggestion
  &__content-part {
    padding: 0 30px;
    margin-bottom: 116px;

    @include onTablet {
      margin-bottom: 140px;
    }

    @include onDesktop {
      margin-bottom: 170px;
      padding: 0;
    }
  }
```
It's easier to read, no?

---

Thanks. I was inspired by video lessons and made this type of code which was useful for me)

---

✔️